farmdata2_modules/fd2_tabs/fd2_example/ui.html

```diff
@@ -314,10 +337,10 @@
             this.tableData.push({id: 4, data: ['12', 'Socks', 'L', 'Red', 22, '2020-04-01']})
         },
         fetchSeedingLogs() {
-            // To use the spinner display it
+            // To display the spinner
             this.waitingOnLogs = true
 
-            this.seedingLogs = []
+            //this.seedingLogs = []
```

Comment:
Noticed that I left this code uncommented when it shouldn't have. It will be back in the next commit.